src/Components/ArtworkFilter/ArtworkFilters/FilterExpandable.tsx

```diff
@@ -19,7 +19,7 @@ export const FilterExpandable: React.FC<
       const anchor = ref.current
       if (!anchor) return
       const { top } = anchor.getBoundingClientRect()
-      if (top < window.innerHeight) return
+      if (top < window.innerHeight && !ignoreBounds) return
```

Comment:
The sticky button breaks the calculation and it should not be take in to account here. We didn't notice this earlier because previous sections were big so it didn't matter as much. I am open to suggestions for a better fix here

Reply:
good idea